nbi/buildspec.yml

```diff
@@ -31,6 +29,7 @@ phases:
       - conda config --set path_conflict warn
       - conda env create --name $BRAKET_ENV -f environment.yml --no-default-package --solver libmamba
       - pip cache purge
+      - du -h -d 4 /root/Braket/miniconda3/envs/$BRAKET_ENV
```

Comment:
Why are you adding this?

Reply:
This makes memory profiling a bit easier in case we even want to look at what is taking up space in the env.

Reply:
We already have a rough summary output so this gives a better insight into particular folders/packages that are raising the install mem requirements. If it causes too much noise, this can be removed as it is not crucial for debugging.